ui/src/store.js

```diff
@@ -1,26 +1,17 @@
-import { createStore, applyMiddleware, compose } from 'redux';
-import { createLogger } from 'redux-logger';
+import { createStore, applyMiddleware } from 'redux';
 import thunk from 'redux-thunk';
 import rootReducer from './reducers';
+import { composeWithDevTools } from '@redux-devtools/extension';
 
-function initStore() {
-  // Logger MUST BE the last middleware
-  const middleware = [thunk, createLogger()];
+export const store = createStore(
+  rootReducer,
+  composeWithDevTools(applyMiddleware(thunk)),
```

Comment:
The `@redux-devtools/extension` package makes things much simpler: https://github.com/reduxjs/redux-devtools/tree/main/extension#13-use-redux-devtoolsextension-package-from-npm